### ISSUE TITLE
Sword of Dungeons and Dragons does not show up when searching by not:reprint

### ISSUE THREAD

**Issue by gold-mir** [scryfall/feedback#731]
Since Sword of D&D's first printing was in the Hascon promo box, and promos are not searched by default, Sword of D&D does not show up at all when trying to search while filtering by original printing.

**Comment by silasary** [comment]
Sword of D&D falls into the same weird category as September Duel Deck
Preprints, Prerelease Promos, and certain Media Inserts.

It is categorized as a promotional "reprint" from a then upcoming set.

**Comment by gold-mir** [comment]
Yeah. There seems to be something different about Sword of D&D specifically though, because it is (as far as I can tell) the only unique card that doesn't show up when searching by original printings. Other "pre-reprinted" cards are either shown by default when searching (Duel Deck printings, for example), or are media inserts that are searched anyway (like Mana Crypt).

I can't find any other pre-reprinted cards that won't find a single printing when searching by not:reprint. Compare [this search for 0 cmc mana artifacts](https://scryfall.com/search?q=not%3Areprint+t%3Aartifact+cmc%3A0+o%3Amana&unique=cards&as=grid&order=name) which *does* find Mana Crypt despite it's first printing being a media insert, to [this search for swords](https://scryfall.com/search?q=%22sword+of%22+not%3Areprint+cmc%3A3+o%3Aequip+o%3A%7B2%7D&unique=cards&as=grid&order=name) which fails to find Sword of Dungeons and Dragons.

**Comment by NilsEnevoldsen** [comment]
I don't see the issue. The original printing is an "extra", so it only shows up when searching with `include:extras` or `is:extra`. The media inserts are not "extras".

I believe @silasary is incorrect that the Hascon promo is "a promotional 'reprint' from a then upcoming set." In fact, the Hascon promo is _not_ a reprint, as you can see by searching `!"sword of dungeons & dragons" not:reprint`.

**Comment by jhallbachner** [comment]
Yeah this is just falling afoul of the extras gate. It's true that this is the only card to which this applies, due to the specifics of how pre-prints and extras work. Actual Un-sets are not considered extras, but unique silver-bordered promos are. Other "pre-prints" have all appeared in Duel Decks or FTV sets which are considered standard reprint products and therefore not extras. As such, Sword of Dungeons and Dragons is the only card in existence which has been printed in a non-extra set but whose initial printing was considered an extra.

**Comment by NilsEnevoldsen** [comment]
Working as intended. Okay to close?

**Comment by silasary** [comment]
From maro:

> It was Sword of Dungeons & Dragons in Unstable, previewed in the Hascon cards. [1]
> It was created in Unstable design. [2]

[1] http://markrosewater.tumblr.com/post/165441004738/last-year-you-said-we-hadnt-seen-your-favorite
[2] http://markrosewater.tumblr.com/post/163705455348/i-heard-ken-nagle-designed-hescon-promo-grimlock

**Comment by NilsEnevoldsen** [comment]
Oh, gotcha. I thought you meant Scryfall categorizes it as a reprint. You meant that it _is_ a reprint, but Scryfall categorizes it incorrectly.
